### PR TITLE
Support idle and fps options

### PR DESCRIPTION
**Pull request by ngocdaothanh**
See https://github.com/marionebl/svg-term/issues/13

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -5,4 +5,4 @@
 [submodule "vt"]
 	path = vt
 	url = https://github.com/asciinema/vt.git
-	branch = e73c392e1aef25144231de4f5f87d72463ca9ccd
+	branch = 83bf644ee1f36ffc5a5ee4c5d0185cc1b1b222db
```
Comment (ngocdaothanh) [inline .gitmodules:8]: Latest version

File: README.md [hunk 2]
```diff
@@ -23,7 +23,17 @@ const read = promisify(fs.readFile);
...
+  // Optional;
+  // if width, height, and idle are not specified, they will be taken from data
+  const options = {
+    width: 80,  // Screencast width
+    height: 25, // Screencast height
+    idle: 2.5,  // Idle time limit in seconds
+    fps: 30     // Frames per second limit, see https://github.com/marionebl/svg-term/issues/13
+  };
+
+  const asciicast = await load(data, options);
```
Comment (ngocdaothanh) [inline README.md:36]: Changed signature

File: package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "load-asciicast",
-  "version": "1.0.2",
+  "version": "2.0.0",
```
Comment (ngocdaothanh) [inline package.json:3]: New major version; please publish to NPM

File: package.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
   "homepage": "https://github.com/marionebl/load-asciicast#readme",
   "devDependencies": {
     "@marionebl/sander": "^0.6.1",
-    "ava": "^0.23.0",
-    "pkg-dir": "^2.0.0"
+    "ava": "^2.4.0",
+    "pkg-dir": "^4.2.0"
```
Comment (ngocdaothanh) [inline package.json:28]: Latest versions

File: project.clj [hunk 5]
```diff
@@ -4,12 +4,12 @@
...
-                 [org.clojure/clojurescript "1.9.671"]
-                 [org.clojure/core.match "0.3.0-alpha4"]
-                 [prismatic/schema "1.1.6"]]
+  :dependencies [[org.clojure/clojure "1.10.1"]
+                 [org.clojure/clojurescript "1.10.520"]
+                 [org.clojure/core.match "0.3.0"]
+                 [prismatic/schema "1.1.12"]]
 
-  :plugins [[lein-cljsbuild "1.1.5"]]
+  :plugins [[lein-cljsbuild "1.1.7"]]
```
Comment (ngocdaothanh) [inline project.clj:12]: Latest versions

**Review by marionebl — APPROVED** [review]
Thanks <3

**Review by marionebl — APPROVED** [review]
Thanks <3

**Comment by marionebl** [comment]
Published as v2.0.0: 

```
❯ npm info load-asciicast

load-asciicast@2.0.0 | MIT | deps: none | versions: 5
Load, normalize and init asciinema asciicast data
https://github.com/marionebl/load-asciicast#readme

keywords: asciinema, asciicast

dist
.tarball: https://registry.npmjs.org/load-asciicast/-/load-asciicast-2.0.0.tgz
.shasum: 298f682d3387d76c48b59d800178fee05d45f1a3
.integrity: sha512-ZS43VhgitBJ4L9+iNT3T89RHrqqhE8+LSC1md4QYYPjsPYiiEjshb19Doxrm0C+YwALlGcem0HhtIYrMG/XQCw==
.unpackedSize: 1.3 MB

maintainers:
- marionebl <hello@mario-nebl.de>

dist-tags:
latest: 2.0.0

published 55 seconds ago by marionebl <hello@mario-nebl.de>
```

**Pull request merged**